src/schemas/complaint-tool/schema.js

```diff
@@ -0,0 +1,165 @@
+    fullName: {
+      '$ref': '#/definitions/fullName'
+    },
+    email: {
+      type: 'string',  // Type: email (no length limit)
+      format: 'email'
+    },
+    educationDetails: {
+      required: [ // no fields are required for submission, though several are required by design on FE
+        // 'school',
+        // 'programs'
+      ],
+      school: definitions.school,
+      programs: { // TODO: Needs to be translated into a (comma) delimited string, and spelled out if within 255 limit 
+        type: 'object',
+        properties: {
+          'MGIB-AD Ch 30': {
+            type: 'boolean'
+            // title: 'Montgomery GI Bill - Active Duty (MGIB) (Ch. 30)'
+          },
+          'MGIB-SR Ch 1606': {
+            type: 'boolean'
+            // title: 'Montgomery GI Bill - Selected Reserve (MGIB-SR) (Ch. 1606)'
+          },
+          'VRE Ch 31': {
+            type: 'boolean'
+            // title: 'Vocational Rehabilitation and Employment (VR&E) (Ch. 31)'
+          },
+          'Post- 9/11 Ch 33': {
+            type: 'boolean'
+            // title: 'Post-9/11 GI Bill (Ch. 33)'
+          },
+          'DEA Ch 35': {
+            type: 'boolean'
+            // title: 'Survivors & Dependents Assistance (DEA) (Ch. 35)'
+          },
+          TATU: {
+            type: 'boolean'
+            // title: 'Tuition Assistance Top-Up'
+          }
+        }
+      },
+      assistance: { // TODO: Needs to be translated into a (comma) delimited string (255 limit), abbr provided to support content changes
+        type: 'object',
+        properties: {
+          TA: {
+            type: 'boolean'
+            // title: Federal Tuition Assistance (TA)
+          },
+          'TA-AGR': {
+            type: 'boolean'
+            // title: State Funded Tuition Assistance (TA) for Service members performing Active Guard and Reserve (AGR) duties 
+          },
+          MyCAA: {
+            type: 'boolean'
+            // title: State Funded Tuition Assistance (TA) for Service members performing Active Guard and Reserve (AGR) duties 
+          },
+          FFA: {
+            type: 'boolean'
+            // title: Federal Financial Aid 
+          }
+        }
+      }
+    },
+    issue: { 
+      type: 'string',
+      'enum': [
+        'Recruiting/Marketing Practices',
+        'Accreditation',
+        'Financial Issues (e.g. Tuition/Fee Charges)',
+        'Student Loans',
+        'Post-Graduation Job Opportunities',
+        'Change in degree/plan requirements',
+        'Quality of Education',
+        'Grade Policy',
+        'Release of Transcripts'
+      ]
+    },
+    issueDescription: {
+      type: 'string',
+      maxLength: 32000
+    },
+    issueResolution: {
+      type: 'string',
+      maxLength: 1000
+    }
+  }
+};
+
+[
+  ['privacyAgreementAccepted'],
+  ['usaPhone', 'phone'], // TODO: note validation requirements pending feedback
+  ['address'], // TODO: note length limits pending feedback
```

Comment:
did we get an answer about international addresses? seems like we should use the same address definition that's used in the vic schema

Reply:
Actually, only US addresses will be accepted, both the school and the applicant address have different validations as well.